android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -180,42 +176,41 @@ private void showLocationsOnMap() {
         });
     }
 
-    private void focucOnMarker(Marker marker){
-        marker.showInfoWindow();
```

Comment:
The thing is cluster manager does not in any way return the reference of the markers created from the location items it is passed to. The markers created inside the class gets stored in a cache and gets rendered as per the view of the cluster. Therefore it never returned the full list of markers hence crashing the app. It is unfortunately something that remains to be implemented by it. For the time being what I have done is just focus the camera on the particular marker since that only requires the location coordinates. Hope I am clear.